python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -197,6 +197,18 @@ def launch_run(self, context: LaunchRunContext) -> None:
             "family"
         ]
 
+        # ECS limits overrides to 8192 characters including json formatting
+        # https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_RunTask.html
+        # When container_context is serialized as part of the ExecuteRunArgs, we risk
+        # going over this limit (for example, if many secrets have been set). This strips
+        # the container context off of our pipeline origin because we don't actually need
+        # it to launch the run; we only needed it to create the task definition.
+        repository_origin = pipeline_origin.repository_origin
+        # pylint: disable=protected-access
+        stripped_repository_origin = repository_origin._replace(container_context={})
+        pipeline_origin = pipeline_origin._replace(repository_origin=stripped_repository_origin)
```

Comment:
lets have a different var instead of overwriting `pipeline_origin` to be explicit